loki/transform/dependency_transform.py

```diff
@@ -177,6 +184,20 @@ def rename_imports(self, source, imports, **kwargs):
         targets = as_tuple(kwargs.get('targets', None))
         targets = as_tuple(str(t).upper() for t in targets)
 
+        # We don't want to rename module variable imports, so we build
+        # a list of calls to further filter the targets
+        if isinstance(source, Module):
+            calls = ()
+            for routine in source.subroutines:
+                calls += as_tuple(str(c.name).upper() for c in FindNodes(CallStatement).visit(routine.body))
+                calls += as_tuple(str(c).upper() for c in FindInlineCalls().visit(routine.body))
+        else:
+            calls = as_tuple(str(c.name).upper() for c in FindNodes(CallStatement).visit(source.body))
+            calls += as_tuple(str(c).upper() for c in FindInlineCalls().visit(source.body))
+
+        # Import statements still point to unmodified call names
+        calls = [call.replace(f'{self.suffix.upper()}', '') for call in calls]
```

Comment:
We don't want to rename the global variables themselves or the modules they are declared in because they are probably used throughout non-device code, and we want to keep those variables coherent with the ones used on device. But yes, let's please discuss offline.